src/components/onboarding/screens/choose-account.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Screen, ScreenBody } from '@blockstack/connect';
+import { Box } from '@blockstack/ui';
+import { ScreenHeader } from '@components/connected-screen-header';
+import { Accounts } from '@components/accounts';
+import { AppIcon } from '@components/app-icon';
+import { useSelector } from 'react-redux';
+import { AppState } from '@store';
+import { selectAppName } from '@store/onboarding/selectors';
+import { Drawer } from '@components/drawer';
+import { selectIdentities } from '@store/wallet/selectors';
+import Identity from '@blockstack/keychain/dist/identity';
+
+interface ChooseAccountProps {
+  next: (identityIndex: number) => Promise<void>;
+  back?: () => void;
+}
+
+export const ChooseAccount: React.FC<ChooseAccountProps> = ({ next }) => {
+  const { appName, identities } = useSelector((state: AppState) => ({
+    appName: selectAppName(state),
+    identities: selectIdentities(state) as Identity[],
```

Comment:
Well the selector can return null - except that we know this component never gets rendered unless `identities` is present.

I'm also happy to add an `if (!identities) { }` check below this so that the compiler doesn't need this flag.

Reply:
Default store state can be annoying. 

In this instance, can identities slice of state default be an empty array, then we avoid the `null` altogether?